.woodpecker/.release.yml

```diff
@@ -0,0 +1,15 @@
+clone:
+  git:
+    image: plugins/git:next
+
+pipeline:
+  release:
+    image: ghcr.io/goreleaser/goreleaser:v0.174.1
+    secrets: [github_token, docker_username, docker_password]
+    commands:
+      - release
+    when:
+      event: tag
```

Comment:
@6543 I think goreleaser uses the tag name as version for tags and bumps the version if executed on pushes based on the conventional commits since the last version? Wouldn't that break the release flow?

If I understand you correctly, you want to release snapshots / pre-releases on master and the official releases on tags?

Reply:
What do you think about continuous delivery with semantic-versioning based on conventional commits? The tool [go-semantic-release](https://github.com/go-semantic-release/semantic-release) looks quite nice for that. We could then stay with the current way of building docker images, binaries via the makefile and build the packages (apk, deb, rpm, ...) with the goreleasers interal tool [nfpm](https://nfpm.goreleaser.com/). As quite some features of goreleaser are only available with the pro edition, this could be a nice and free solution.

Reply:
Okay I changed the release flow a bit so it should create a `next` image for pushes to master and create the appropriate github release with binaries, checksums, changelog etc for tags / releases.